$NetBSD: patch-src_perl_tabbed,v 1.1 2013/07/22 17:55:21 morr Exp $

Fix to make it compatibible with perl 5.18 
(https://metacpan.org/module/RJBS/perl-5.18.0/pod/perldelta.pod#qw-...-can-no-longer-be-used-as-parentheses)

Patch taken from http://lists.schmorp.de/pipermail/rxvt-unicode/2013q2/001793.html

--- src/perl/tabbed.orig	2013-02-17 00:41:38.000000000 +0000
+++ src/perl/tabbed
@@ -402,7 +402,7 @@ package urxvt::ext::tabbed::tab;
 # simply proxies all interesting calls back to the tabbed class.
 
 {
-   for my $hook qw(start destroy key_press property_notify) {
+   for my $hook (qw(start destroy key_press property_notify)) {
       eval qq{
          sub on_$hook {
             my \$parent = \$_[0]{term}{parent}
